raven/src/main/java/com/getsentry/raven/util/Util.java

```diff
@@ -65,4 +65,22 @@ public static boolean isNullOrEmpty(@Nullable String string) {
         return new HashSet<>(Arrays.asList(extraTagsString.split(",")));
     }
 
+    /**
+     * Parses the provided string value into integer value.
+     * <p>If parsing can't be performed then return default value.</p>
+     *
+     * @param value        value to parse
+     * @param defaultValue default value
+     * @return integer representation of provided value or default value.
+     */
+    public static int parseInteger(String value, int defaultValue) {
+        if (isNullOrEmpty(value)) {
+            return defaultValue;
+        }
+        try {
+            return Integer.parseInt(value);
+        } catch (NumberFormatException e) {
+            return defaultValue;
```

Comment:
Good catch. I actually think maybe it should blow up. IMO the default should only be used when the input is blank/null.